x/concentrated-liquidity/README.md

```diff
@@ -1582,6 +1582,117 @@ Note that for storing ticks, we use 9 bytes instead of directly using uint64, fi
 Note that the reason for having pool ID and min uptime index is so that we can retrieve
 all incentive records for a given pool ID and min uptime index by performing prefix iteration.
 
+## Precision Issues With Price
+
+There are precision issues that we must be considerate of in our design.
+
+Consider the balancer pool between `uarb` and `uosmo`:
+
+```bash
+osmosisd q gamm pool 1011
+pool:
+  '@type': /osmosis.gamm.v1beta1.Pool
+  address: osmo1pv6ffw8whyle2nyxhh8re44k4mu4smqd7fd66cu2y8gftw3473csxft8y5
+  future_pool_governor: 24h
+  id: "1011"
+  pool_assets:
+  - token:
+      amount: "101170077995723619690981"
+      denom: ibc/10E5E5B06D78FFBB61FD9F89209DEE5FD4446ED0550CBB8E3747DA79E10D9DC6
+    weight: "536870912000000"
+  - token:
+      amount: "218023341414"
+      denom: uosmo
+    weight: "536870912000000"
+  pool_params:
+    exit_fee: "0.000000000000000000"
+    smooth_weight_change_params: null
+    swap_fee: "0.002000000000000000"
+  total_shares:
+    amount: "18282469846754434906194"
+    denom: gamm/pool/1011
+  total_weight: "1073741824000000"
+```
+
+Let's say we want to migrate this into a CL pool where `uosmo` is the quote
+asset and `uarb` is the base asset.
+
+Note that quote asset is denom1 and base asset is denom0.
+We want quote asset to be `uosmo` so that limit oredrs on ticks
+have tick spacing in terms of `uosmo` as the quote.
+
+
+Note:
+- OSMO has precision of 6. 1 OSMO = 10**6 `uosmo`
+- ARB has precision of 18. 1 ARB = 10**18 `uarb`
```

Comment:
Would this technically be `uarb`? My understanding was that the `u` represented µ/micro/10^-6 (might be mistaken on this lol)

Reply:
My interpretation of `u` was the base AMM unit. However, I'm not sure so I will change this to something else

Reply:
Alpin is correct in it referring to micro

Reply:
addressed